docker-compose.yaml

```diff
@@ -70,7 +70,7 @@ services:
       POSTGRES_PASSWORD: jimm
     # Since it's mainly used for testing purposes, it's okay to set fsync=off for
     # improved performance.
-    command: -c fsync=off -c full_page_writes=off
+    command: -c fsync=off -c full_page_writes=off -c max_connections=200
```

Comment:
This is a bit concerning. I wonder if we're going over the default 100 connections because we're leaking connections or because of tests running in parallel. Perhaps one way to check would be to add a global test cleanup methods that checks the number of open connections to Postgres. Or run tests with -p 1 and print the number of connections to Postgres between each test incovation.